### PR TITLE
Continued Work on database setup: elastic beanstalk

### PR DESCRIPTION
**Pull request by PaulMEdwards**
# Description

~~THIS WORK IS INCOMPLETE/IN-PROGRESS (hence it being a DRAFT Pull Request)...~~
~~Please DO NOT Review/Merge until it has been marked as READY! 👹~~

This collection of changes automates the following on the AWS EC2 environment:
- Create the `.env` file from the AWS Environment Software Configuration settings.
- Install `psql` via yum package manager.
- Execute 3 SQL commands to ensure the database user is created, the database is created, and the user has permissions to the database. These are the same SQL commands run in the local Development Docker container using the `postgres-docker.bash` script.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x] Tested manually by logging in to AWS / ElasticBeanstalk CLI and inspecting the deployed/created files as well as inspecting the database contents using pgAdmin 4.

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Comment by PaulMEdwards** [comment]
[Get Production Backend working in AWS](https://trello.com/c/1xsq6dzC/60-get-production-backend-working-in-aws)

**Review by Aaroneld — APPROVED** [review]
Thank you for fixing this environment issue 🌳 I bet it was pernicious

**Comment by PaulMEdwards** [comment]
We can squish merge this into `master` when it is approved.

File: .elasticbeanstalk/config.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+branch-defaults:
```
Comment (PaulMEdwards) [inline .elasticbeanstalk/config.yml:1]: This file is created by the ElasticBeanstalk CLI when running `eb init` on the project directory. It contains settings used by the `eb` command to connect to the desired AWS EC2 instance.

File: .gitignore [hunk 2]
```diff
@@ -118,3 +118,8 @@ yarn.lock
 
 #Local docker data files
 data/docker
+
```
Comment (PaulMEdwards) [inline .gitignore:121]: These were added by the ElasticBeanstalk CLI when running `eb init`.

File: package.json [hunk 3]
```diff
@@ -8,7 +8,7 @@
     "test:watch": "cross-env NODE_ENV=development NO_LOGGER=true jest --watch --runInBand --detectOpenHandles --coverage",
     "test:watchTroubleshoot": "cross-env NODE_ENV=development NO_LOGGER=true jest --watch --runInBand --detectOpenHandles",
     "test:watchWithLogs": "cross-env NODE_ENV=development jest --watch --runInBand --detectOpenHandles",
-    "prestart": "knex migrate:latest",
+    "prestart": "npx knex migrate:latest",
```
Comment (PaulMEdwards) [inline package.json:11]: Need to use `npx` because `knex` isn't in the system path.

**Review by kyle-richardson — APPROVED** [review]
woohoo! we got a deployed database! well done. *bows*

**Pull request merged**